conda.recipe/post-link.bat

```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
+"%PREFIX%\python" -m nb_conda.setup install --enable --prefix="%PREFIX%"
```

Comment:
Should not be `"%PREFIX%\python.exe"`... btw, how it is possible that the python exe on win is not living in Scripts and lives in prefix???